README.md

```diff
@@ -70,6 +70,67 @@ If you want to limit the amount of inferred unstable classes, you can configure
 This will make the `./gradlew verifyComposeMetrics` task fail if the module has crossed the threshold. If you just want it to print a warning, you can 
 use the `errorAsWarning` flag and set it to `true`.
 
+### Example
+
+If you have a screen with composable functions like this:
+
+```kt
+@Composable
+fun Screen() {
+    val state = ScreenState.Content()
+    SubScreen(state)
+}
+
+@Composable
+fun SubScreen(state: ScreenState) {
+    Text(text = state.toString())
+    val unstableState = UnstableClass()
+    UnstableScreen(unstableClass = unstableState)
+}
+
+@Composable
+fun UnstableScreen(unstableClass: UnstableClass) {
+    Text(text = unstableClass.greeting)
+}
+
+sealed class ScreenState {
+    data class Content(val greeting:String = "Hey! This class Should be immutable"): ScreenState()
+}
+
+data class UnstableClass(
+    var greeting: String = "Hey this makes me an unstable type!"
+)
+
+data class UnstableGreetingsClass(
+    var greeting: String = "Hey this makes me an unstable type!"
+)
+
+```
+And then adding a verifyComposeMetrics configuration like this:
```

Comment:
`And then adding a verifyComposeMetrics configuration into build.gradle.kts like this:`

Reply:
I agree! Updated it in [5fbe009](https://github.com/oas004/VerifyComposeMetricsPlugin/pull/6/commits/5fbe009411558cb1fbc8cdd8ba334df6a1e2daa6) Thanks! 😄